src/screens/HomeStackScreen.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * This component is the navigation flow for an authenicated user (logged in)
+ */
+
+import React from "react";
+import { createStackNavigator } from "@react-navigation/stack";
+
+import theme from "../theme";
+import { ThemedIcon } from "../components/themed";
+import MeetupResponsePage, {
+  MeetupResponsePageOptions,
+} from "../components/meetups/MeetupResponsePage";
+import MeetupDetailsPage, {
+  MeetupDetailsPageOptions,
+} from "../components/meetups/MeetupDetailsPage";
+import MeetupListPage, {
+  MeetupListPageOptions,
+} from "../components/meetups/MeetupListPage";
+import MeetupRatingPage, {
+  MeetupRatingPageOptions,
+} from "../components/meetups/MeetupRatingPage";
+
+const HomeStack = createStackNavigator();
+
+const HomeStackScreen = (
+  { navigation }: any // eslint-disable-line @typescript-eslint/no-explicit-any
+) => {
+  return (
+    <HomeStack.Navigator
+      screenOptions={{
+        headerStyle: {
+          backgroundColor: theme.colors.headerBackground,
+        },
+        headerTintColor: theme.colors.primaryText,
+      }}
+    >
+      <HomeStack.Screen
+        name="Home"
+        component={MeetupListPage}
+        options={{
+          ...MeetupListPageOptions,
+          headerLeft: () => {
+            return (
+              <ThemedIcon
+                size={24}
+                reverse
+                name="menu"
+                type="entypo"
+                onPress={() => {
+                  navigation.openDrawer();
+                }}
+                color={theme.colors.headerBackground}
+                iconStyle={{ color: theme.colors.primaryText }}
+              />
+            );
+          },
+        }}
+        initialParams={{
+          futureMeetupIDs: ["test"],
+          tentativeMeetupIDs: ["temp"],
+        }}
+      />
+
+      <HomeStack.Screen
+        name="MeetupResponse"
+        component={MeetupResponsePage}
+        options={MeetupResponsePageOptions}
+      />
+
+      <HomeStack.Screen
```

Comment:
Is there a reason that certain screens are duplicated in HomeStack and MeetupStack ?

Reply:
yes there was an initial version where i separated all the screens out and a new version where I actually pieced things together. all in all, I dont want to delete anything until the final flow is completed and all screens are integrated